### PR TITLE
Aden 5134

### PR DESCRIPTION
**Pull request by bulaszenko**

### CODE REVIEW

**Comment by WikiaTech** [comment]
SonarQube analysis reported 3 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [AdsDataProvider.java#L1275](https://github.com/Wikia/selenium-tests/blob/79d788c19f27e4219e8eb03382e6054b66b86d78/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L1275): Move this trailing comment on the previous empty line. [rule](http://sonar-s4/coding_rules#rule_key=squid%3ATrailingCommentCheck)
1. ![MINOR][MINOR] [AdsOoyalaObject.java#L65](https://github.com/Wikia/selenium-tests/blob/79d788c19f27e4219e8eb03382e6054b66b86d78/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsOoyalaObject.java#L65): Define a constant instead of duplicating this literal " during " 4 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [AdsOoyalaObject.java#L66](https://github.com/Wikia/selenium-tests/blob/79d788c19f27e4219e8eb03382e6054b66b86d78/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsOoyalaObject.java#L66): Define a constant instead of duplicating this literal " seconds" 4 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [AdsDataProvider.java#L1275](https://github.com/Wikia/selenium-tests/blob/79d788c19f27e4219e8eb03382e6054b66b86d78/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L1275): Move this trailing comment on the previous empty line. [rule](http://sonar-s4/coding_rules#rule_key=squid%3ATrailingCommentCheck)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsMonocolor.java [hunk 1]
```diff
@@ -1,23 +0,0 @@
...
-
-  @Test(
-      groups = "AdsMonocolorOasis",
-      dataProviderClass = AdsDataProvider.class,
-      dataProvider = "adsMonocolorOasis"
-  )
-  public void adsMonocolorOasis(String wikiName, String pageName) {
-    String testPage = urlBuilder.getUrlForPath(wikiName, pageName);
-    AdsBaseObject adsBaseObject = new AdsBaseObject(driver, testPage);
-    adsBaseObject.verifyMonocolorAd(AdsContent.TOP_LB);
```
Comment (fraszczakszymon) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsMonocolor.java]: Most likely we can remove `verifyMonocolorAd` as well

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [AdsDataProvider.java#L1275](https://github.com/Wikia/selenium-tests/blob/871544dcacadb0bdacf5e87e9e7aafcb12fc2feb/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L1275): Move this trailing comment on the previous empty line. [rule](http://sonar-s4/coding_rules#rule_key=squid%3ATrailingCommentCheck)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsMonocolor.java (hunk 1 above)
Comment (bulaszenko) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsMonocolor.java]: Done

**Pull request merged**